src/dev.ts

```diff
@@ -38,7 +46,7 @@ export async function startDevWatcher() {
       }
 
       debug('update from', path)
-      await utils!.extractFile(await fs.readFile(path, 'utf-8'))
+      await utils!.extractFile(fs.readFileSync(path, 'utf-8'))
```

Comment:
Why do we need to change this to sync?

Reply:
I chose the sync variant because it seems easier to me and we don't wait for anything.

Reply:
`readFileSync` blocks the entire process when waiting for the fs, it's actually slowing down the entire process. While this is not significant, I don't think we need to change this line if the existing code works fine and there is no improvement. Can you revert this?

Reply:
jep.

Reply:
@antfu Is the async handling better now?

Reply:
- [ ] I think I have to consolidate the touch function.